components/match2/commons/match_group_display_helper.lua

```diff
@@ -211,6 +211,16 @@ function DisplayHelper.DefaultMatchSummaryContainer(props)
 	return MatchSummaryModule.getByMatchId(props)
 end
 
+---@param props table
+---@return Html
+function DisplayHelper.DefaultGameSummaryContainer(props)
+	local GameSummaryModule = Lua.import('Module:GameSummary')
+
+	assert(GameSummaryModule.getGameByMatchId, 'Expected GameSummary.getGameByMatchId to be a function')
```

Comment:
if we say in the error we expect it to be a function, **maybe** check for it being a function instead of just existing
```suggestion
	assert(type(GameSummaryModule.getGameByMatchId) = 'function', 'Expected GameSummary.getGameByMatchId to be a function')
```